src/core/cardsList/CardsList.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+import Grid from '../../common/components/grid/Grid'
+
+export type CardListProps = {
+  children: React.ReactNode
+}
+
+export default function CardsList({ children }: CardListProps) {
```

Comment:
I wouldn't mix the "Lists" and "Grids" (the terms) together. Couldn't this be a CardGrid instead? Or since this is a generic component lib, do we need a specific CardGrid at all? Couldn't we just share a code script as an example?

Reply:
I know the MUI has the ImageList (https://mui.com/material-ui/api/image-list/), but I find that confusing also, because it's clearly a grid with a fixed amount of columns.

Reply:
Okay, it could be a list if the heights are not fixed and it works like this: https://nuorten.hel.fi/osallistu-ja-vaikuta/ 
![image](https://user-images.githubusercontent.com/389204/167564223-92dc8c48-c76e-43f7-bac0-9efcad98614a.png)

Reply:
yes, i ll rename. Makes sense